src/controllers/login.controller.js

```diff
@@ -8,27 +8,25 @@ module.exports = {
     try {
       let user;
       let typeOf;
-       user = await Provider.findOne({ providerEmail: req.body.email });
-      if (!user){
+      user = await Provider.findOne({ providerEmail: req.body.email });
+      if (!user) {
         user = await Client.findOne({ clientEmail: req.body.email });
         typeOf = 'client';
-        }
-      if (!user){
-        console.log(user);
+      }
+      if (!user) {
+        // console.log(user);
         throw Error('El usuario no existe');
       }
       const isValid = await bcrypt.compare(req.body.password, user.password);
       if (!isValid) {
         throw Error('Usuario o contraseña invalida');
-        }
-      const token = jwt.sign(
-        { id: user._id },
-        process.env.SECRET,
-        { expiresIn: 60 * 60 * 24 * 365 }
-        );
+      }
+      const token = jwt.sign({ id: user._id }, process.env.SECRET, {
+        expiresIn: 60 * 60 * 24 * 365,
+      });
       res.status(200).json({ token, typeOf });
-     } catch (error) {
-      res.status(401).json({ message: error.message });
+    } catch (error) {
+      res.status(400).json({ message: error.message });
```

Comment:
Por qué el cambio a 400? recuerda que cada código tiene un significado, en este caso 401 Not Authorized

Reply:
En este caso sería el 401 ?

Reply:
Ya realicé los cambios, quedo atenta ante las correcciones :D